src/components/posts/AddPost.tsx

```diff
@@ -0,0 +1,141 @@
+import { joiResolver } from "@hookform/resolvers/joi";
+import axios from "axios";
+import { baseURL } from "@config";
+import { PostActionsEnums } from "@enums";
```

Comment:
We should make a task where you'll implement some eslint rules for cleaner imports and code in general.